app/scripts/modules/ecs/src/serverGroup/details/events/serverGroupEventsReader.service.ts

```diff
@@ -0,0 +1,25 @@
+import { IPromise } from 'angular';
+
+import { $log } from 'ngimport';
+
+import { API } from '../../../../../core/src/api/ApiService';
+import { IServerGroup } from '../../../../../core/src/domain/index';
+
+export class ServerGroupEventsReader {
+  public static getEvents(serverGroup: IServerGroup): IPromise<any[]> {
```

Comment:
I would strongly type the return value of this method (i.e., `IEventDescription` should be defined in this file and the return type here should be `IPromise<IEventDescription[]>`).